lintreview/review.py

```diff
@@ -104,11 +101,13 @@ class Review(object):
     to github.
     """
 
-    def __init__(self, gh, number):
+    def __init__(self, gh, number, config=None):
+        config = config if config is not None else {}
```

Comment:
Sorry, I forgot to point out that this can also be `config = config if config else {}`.
My bad!